Xamarin.Forms.Platform.Android/Renderers/LabelRenderer.cs

```diff
@@ -332,5 +336,18 @@ public override bool OnTouchEvent(MotionEvent e)
 
 			return _motionEventHelper.HandleMotionEvent(Parent, e);
 		}
+
+		void UpdateBreakStrategy()
+		{
+			if (Element == null || Control == null)
+				return;
+
+			if (Forms.SdkInt >= BuildVersionCodes.M)
```

Comment:
Can we refactor this code to another place, to avoid [code duplication](https://github.com/xamarin/Xamarin.Forms/pull/11345/files#diff-5a3c2cb189554ccb4f10cc46213bb1adR472)?

Maybe we can transform this into an extension method, and add it [here ](https://github.com/xamarin/xamarin.forms/blob/main/Xamarin.Forms.Platform.Android/Extensions/TextViewExtensions.cs)

Reply:
cc @samhouts